releases/release-6.3.0.md

```diff
@@ -0,0 +1,313 @@
+---
+title: TiDB 6.3.0 Release Notes
+---
+
+# TiDB v6.3.0 Release Notes
+
+Release date: xx xx, 2022
+
+TiDB version: 6.3.0-DMR
+
+In v6.3.0-DMR, the key new features and improvements are as follows:
+
+- TiKV/TiFlash 静态加密支持国密算法 SM4
+- TiDB 支持基于国密算法 SM3 插件的身份验证
+- SQL语句CREATE USER / ALTER USER支持ACCOUNT LOCK/UNLOCK 选项
+- JSON 数据类型和 JSON 函数 GA
+- TiDB 支持 Null Aware Anti Join
+- 提供“执行时间”的细粒度指标
+- 分区表新增简化 Range 分区的语法糖衣，避免在 DDL 中枚举所有分区
+- Range Columns 分区方式在 PARTITION BY RANGE COLUMNS (column_list) 处支持定义多列
+
+## New features
+
+### SQL
+
+* Add a new syntactic sugar to simplify Range partition definitions [#issue]() @[mjonss](https://github.com/mjonss)
+
+    [Provides INTERVAL partitioning as a new way of defining Range partitions](/partitioned-table.md#range-interval-partitioning). You do not need to enumerate all partitions, which drastically reduces the lengthy way of writing Range partition statements. The semantic is equivalent to the original Range partition.
+
+* Range Columns partitioning supports defining multiple columns [#issue]() @[mjonss](https://github.com/mjonss)
+
+    Support [PARTITION BY RANGE COLUMNS (column_list)](/partitioned-table.md#range-columns-partitioning). `column_list` is no longer limited to a single column. The basic feature is the same as MySQL.
+
+* EXCHANGE PARTITION becomes GA [#35996](https://github.com/pingcap/tidb/issues/35996) @[ymkzpx](https://github.com/ymkzpx)
+
+    [EXCHANGE PARTITION]((/partitioned-table.md#partition-management) becomes GA after performance and stability improvements.
+
+* 增加支持以下[窗口分析函数](/tiflash/tiflash-supported-pushdown-calculations.md)：[#5579](https://github.com/pingcap/tiflash/issues/5579) @[SeaRise](https://github.com/SeaRise) **tw：shichun-0415**
+
+    * `LEAD`
+    * `LAG`
+
+* The `CREATE USER` statement supports the `ACCOUNT LOCK/UNLOCK` option [#37051](https://github.com/pingcap/tidb/issues/37051) @[CbcWestwolf](https://github.com/CbcWestwolf) **tw：ran-huang**
+
+    When you create a user using the [`CREATE USER`](/sql-statements/sql-statement-create-user.md) statement, you can specify whether the created user is locked using the `ACCOUNT LOCK/UNLOCK` option. A locked user cannot log in to the database.
+
+* The `ALTER USER` statement supports the `ACCOUNT LOCK/UNLOCK` option [#37051](https://github.com/pingcap/tidb/issues/37051) @[CbcWestwolf](https://github.com/CbcWestwolf) **tw：ran-huang**
+
+    You can modify the lock state of an existing user using the `ACCOUNT LOCK/UNLOCK` option in the [`ALTER USER`](/sql-statements/sql-statement-alter-user.md) statement.
+
+* JSON 数据类型和 JSON 函数 GA [#36993](https://github.com/pingcap/tidb/issues/36993) @[xiongjiwei](https://github.com/xiongjiwei) **tw: qiancai**
+
+    JSON 是一种流行的数据格式，被大量的程序设计所采用。TiDB 在早期版本就引入了 [JSON 支持](/data-type-json.md)， 兼容 MySQL 的 JSON 数据类型 和一部分 JSON 函数。在 v6.3.0 版本中，我们正式将这些功能 GA ，用户可以安全地在生产环境中使用 JSON 相关的功能。 JSON 功能的 GA，为 TiDB 提供了更丰富的数据类型支持，同时也进一步提升的 TiDB 对 MySQL 的兼容能力。
```

Comment:
```suggestion
* JSON data type and JSON functions become GA [#36993](https://github.com/pingcap/tidb/issues/36993) @[xiongjiwei](https://github.com/xiongjiwei) **tw: qiancai**

    JSON is a popular data format adopted by a large number of programs. TiDB has introduced the [JSON support](/data-type-json.md) as experimental since an earlier version, compatible with MySQL's JSON data type and some JSON functions. In v6.3.0, the JSON support becomes GA, providing TiDB with richer data types, and further improving TiDB compatibility with MySQL.
```